spec/features/visitor_can_create_account_spec.rb

```diff
@@ -0,0 +1,47 @@
+# As a visitor
+#     When I visit "/"
+#     Then I should see a link for "Login"
+#     And when I click link "Create Account"
+#     Then my current page should be "/dashboard"
+#     And I should see a message in the navbar that says "Logged in as SOME_USER"
+#     And I should see my profile information
+#     And I should not see a link for "Login"
+#     And I should see a link for "Logout"
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.